foreman.te

```diff
@@ -231,6 +231,13 @@ miscfiles_manage_generic_cert_files(passenger_t)
 # /tmp/passenger-native-support-XXXXXX/passenger_native_support.so
 can_exec(passenger_t, passenger_tmp_t)
 
+# Logrotate in RHEL7 does not allow signals, RHBZ#1527522
+gen_require(`
+    type logrotate_t;
+    type systemd_unit_file_t;
+')
+allow logrotate_t systemd_unit_file_t:service stop;
```

Comment:
For my understanding, is stop just generic for a signal?

Reply:
Oh this is a left over. It must be same as in `foreman-proxy.te` thanks.